docker-compose.test.yml

```diff
@@ -7,6 +7,7 @@ services:
     command: ["go", "test", "-v", "github.com/appcelerator/amp/data/etcd"]
```

Comment:
That sounds good to me. @chrisccoy @bertrand-quenin @freignat91, what this means is add the services that need to be running to the compose file (and keep up-to-date with your PRs). `sut` should just invoke `make test`. Make sure you add your Go packages to test to the `test` in the `Makefile`.